.github/workflows/pr-assign-team-label.yml

```diff
@@ -0,0 +1,16 @@
+name: Assign PR team labels
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  team-labels:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: calcom/team-label-action@main
+        with:
+          repo-token: ${{ secrets.GH_ACCESS_TOKEN }}
+          organization-name: calcom
+          only-labels: "cal-com"
```

Comment:
only-label not quite working yet: https://github.com/calcom/team-label-action/blob/0d3f80de66445d54370716df2b8e834eb40f75b7/src/main.ts#L30

work in progress